tests/test_association.py

```diff
@@ -302,12 +317,12 @@ def testInvalidDb(self):
         """ Test that the measurement raises the correct error when given an
         improper database.
         """
-        conn = sqlite3.connect(":memory:")
-        cursor = conn.cursor()
-        with self.assertRaises(sqlite3.OperationalError):
-            measureTotalUnassociatedDiaObjects(
-                cursor,
-                metricName='association.numTotalUnassociatedDiaObjects')
+        with sqlite3.connect(":memory:") as conn:
+            cursor = conn.cursor()
+            with self.assertRaises(sqlite3.OperationalError):
```

Comment:
This feels like it's just testing that sqlite3 raises an exception when you give it bad input. I don't understand what functionality it's testing in the code itself, other than exception pass-through.

Reply:
I agree. @morriscb, was there some specific functionality in `ap.verify.measurements` being tested in `testInvalidDb`?